### PR TITLE
pylon detection module

### PR DESCRIPTION
**Pull request by rayjinghaolei**
implemented pylon detection, please review

### CODE REVIEW

File: modules/targetAcquisition/pylonAcquisition.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        # draw out the bounding boxes
+        for (topLeft, botRight) in self.bbox:
+            cv2.rectangle(self.currentFrame, topLeft, botRight, (0, 0, 255), 2)
+        cv2.imshow('img', self.currentFrame)
+        cv2.waitKey(0) 
+        cv2.destroyAllWindows()
+
+
+# Testing
+tracker = PylonAcquisition()
```
Comment (Shrinjay) [inline modules/targetAcquisition/pylonAcquisition.py:94]: This is a good idea but please put this in a separate file. I recommend using pytest, create a /tests/ folder at the root directory and put test scripts in there.

File: modules/targetAcquisition/pylonAcquisition.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from time import sleep
+
+from tensorflow.compat.v1 import ConfigProto
+from tensorflow.compat.v1 import InteractiveSession
+
+config = ConfigProto()
+config.gpu_options.allow_growth = True
+session = InteractiveSession(config=config)
+
+class PylonAcquisition:
```
Comment (Shrinjay) [inline modules/targetAcquisition/pylonAcquisition.py:15]: Only nitpick, please rename this class and the file to targetAcquisition just to keep it all the same.

**Review by Shrinjay — CHANGES_REQUESTED** [review]

File: modules/targetAcquisition/pylonAcquisition.py (hunk 2 above)
Comment (rayjinghaolei) [reply modules/targetAcquisition/pylonAcquisition.py:15]: should I just delete the old targetAcquisition and replace it with this one?

File: modules/targetAcquisition/pylonAcquisition.py (hunk 1 above)
Comment (rayjinghaolei) [reply modules/targetAcquisition/pylonAcquisition.py:94]: can you show me how it works when we meet tonight?

File: modules/targetAcquisition/pylonAcquisition.py (hunk 2 above)
Comment (Shrinjay) [reply modules/targetAcquisition/pylonAcquisition.py:15]: Yes

**Review by Shrinjay — APPROVED** [review]

**Pull request merged**